include/adt/edgeset.h

```diff
@@ -68,6 +68,8 @@ edge_set_transition(const struct edge_set *set, unsigned char symbol,
 size_t
 edge_set_count(const struct edge_set *set);
 
+/* Note: this should actually union src into *dst, if *dst already
```

Comment:
I was tripped up by this, twice (albeit months apart), because I don't think of this as typical 'copy' interface behavior.